sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -850,18 +850,18 @@ trait NoLegacyJDBCError extends JdbcDialect {
 
   override def classifyException(
       e: Throwable,
-      errorClass: String,
+      condition: String,
       messageParameters: Map[String, String],
       description: String,
       isRuntime: Boolean): Throwable with SparkThrowable = {
     if (isRuntime) {
       new SparkRuntimeException(
-        errorClass = errorClass,
+        errorClass = condition,
```

Comment:
Is there a plan to rename the `errorClass` argument to `condition` in `SparkRuntimeException` and `AnalysisException`?

Reply:
Yes